src/renderer/modals/Send/Body.js

```diff
@@ -38,6 +38,10 @@ type OwnProps = {|
     startWithWarning?: boolean,
     recipient?: string,
     amount?: BigNumber,
+    disableBacks?: [string],
```

Comment:
Does it mean disableBacks accepts one string item only? If that's what we want, why not simply string?

Reply:
array of string, have i done the type incorrectly ?

Reply:
ha ok string[], will change